### PR TITLE
Print test arrays in java style

### PR DESCRIPTION
**Pull request by pboyer**
Should fix at least 2 tests associated with the issue identified in #47.

Thanks @sridharxp! Note, this requires some changes to the go runtime test templates, not just the output.

### CODE REVIEW

File: runtime/Go/antlr/utils.go [hunk 1]
```diff
@@ -343,6 +341,33 @@ func EscapeWhitespace(s string, escapeSpaces bool) string {
 	return s
 }
 
+func TerminalNodeToStringArray(sa []TerminalNode) []string {
+	st := make([]string, len(sa))
+
+	for i, s := range sa {
+		st[i] = fmt.Sprintf("%v", s)
```
Comment (willfaught) [inline runtime/Go/antlr/utils.go:348]: Would Sprint also work? It would be simpler.

File: runtime/Go/antlr/utils.go [hunk 2]
```diff
@@ -343,6 +341,33 @@ func EscapeWhitespace(s string, escapeSpaces bool) string {
...
+	}
+
+	return st
+}
+
+func PrintArrayJavaStyle(sa []string) string {
+	var buffer bytes.Buffer
```
Comment (willfaught) [inline runtime/Go/antlr/utils.go:355]: Would it work to do something like `"[" + strings.Join(sa, ", ") + "]"`? It seems simpler to me.

File: runtime/Go/antlr/utils.go [hunk 3]
```diff
@@ -343,6 +341,33 @@ func EscapeWhitespace(s string, escapeSpaces bool) string {
...
+	st := make([]string, len(sa))
+
+	for i, s := range sa {
+		st[i] = fmt.Sprintf("%v", s)
+	}
+
+	return st
+}
+
+func PrintArrayJavaStyle(sa []string) string {
```
Comment (willfaught) [inline runtime/Go/antlr/utils.go:354]: Seems strange to me to refer to Java in the Go runtime. This is only here to work around a test problem. Also, "Print" suggests I/O to me. Would something like StringWithCommas (as in `String()` with commas) work better?

**Comment by willfaught** [comment]
Is there any way to change the test to work with both Java and Go instead? That would be ideal.

File: runtime/Go/antlr/utils.go (hunk 2 above)
Comment (pboyer) [reply runtime/Go/antlr/utils.go:355]: Good point! Why'd I merge this without getting you to review it?

Comment (willfaught) [reply runtime/Go/antlr/utils.go:355]: Happy to :)

**Pull request merged**